whitepaper/chapter01.md

```diff
@@ -0,0 +1,140 @@
+
+<a name="1.4"></a>
+## 1.4 Cloud Native Network Functions
+
+A cloud native network function (CNF) is a cloud native application that implements or facilitates network functionality. A cloud native network function consists of one or more microservices, and has been developed using Cloud Native Principles including immutable infrastructure, declarative APIs and a “repeatable deployment process”.
+
+An example of a simple CNF is a packet filter which implements a single piece of network functionality as a microservice. A firewall is an example of a CNF which may be composed of more than one microservice.
+
+The CNFs themselves, by nature of following cloud native principles, are also composable and can implement and facilitate more complex network functionality if needed.
+
+CNFs and cloud native principles can be used when implementing applications and network infrastructure which needs to meet standards such as [3GPP](https://en.wikipedia.org/wiki/3GPP). For example an [Evolved Packet Core's (EPC)](https://en.wikipedia.org/wiki/System_Architecture_Evolution#Evolved_Packet_Core_(EPC)) [Serving Gateway](https://en.wikipedia.org/wiki/System_Architecture_Evolution#SGW_(Serving_Gateway)) could be implemented as a cloud native application and support all 3GPP requirements (eg. S1 protocol stack) allowing integration with other [EPC services](https://en.wikipedia.org/wiki/System_Architecture_Evolution#Evolved_Packet_Core_(EPC)).
+
+<a name="1.5"></a>
+## 1.5 Cloud Native for Telcos
+
+<a name="1.5.1"></a>
+### 1.5.1 A Brief History of Virtualisation for Telcos
+
+Until the 1990’s, most functionality and applications for telecommunications resided primarily in physical hardware (aka ‘boxes’). While software powered these systems, the software functioned more like firmware and was not easy to abstract and run on multiple platforms in virtual environments.
+
+This fostered a “top-down” approach to telecommunications systems that were imperative rather than declarative. This approach was exceptionally resource intensive and expensive; redundancy literally meant acquiring multiple physical instances of required systems. Not surprisingly, this guiding architectural decision encouraged the tight bundling of numerous services and capabilities into individual physical systems. This resulted not only in inflated infrastructure and provisioning costs but also inefficient hyper-redundancy in system engineering for overlapping capabilities.
+
+<a name="1.5.2"></a>
+### 1.5.2 Software Defined Networking And The Emergence of VNFs
```

Comment:
@tomkivlin , @taylor in my POV it will be better if we add  a brief on ETSI NFV, not for virtualization itself covering the both above points (1.5.1&1.5.2) 

I will assign it to myself and provide it

Reply:
@tomkivlin , I can't assign it to myself, plz support here

Reply:
You will need to create a new issue, and%or create a branch on a fork of this repo. This PR has been merged so you won't be able to do it on here.

Reply:
@ASawwaf it seems like what you are proposing should go into a new chapter or section that talks about ETSI NFV from a historical point of view or alternative approach for implementing telephony platforms and technologies. 

Whatever the case, as @tomkivlin said, this PR is merged.  Please create a github issue for further discussion and/or PR.